src/cli.rs

```diff
@@ -122,6 +127,14 @@ pub struct Ping_ {
     pub nix_file: PathBuf,
 }
 
+/// Stream events from the daemon.
+#[derive(StructOpt, Debug)]
+pub struct StreamEvents_ {
+    #[structopt(short, long, default_value = "all")]
```

Comment:
My suggestion would be to leave out the short option here. If we really think an additional short option would improve UX, we can always add it afterwards. On the other hand, removing it at some later point is a [breaking change](https://github.com/target/lorri/blob/master/MAINTAINERS.md#major-or-minor-release). So my suggestion would be to keep the user interface minimal.

Reply:
That's a good point. I was cribbing from elsewhere, but the plumbing commands don't need to have shortcuts like that.